### DISCUSSION TITLE
请问我如何能在monorepo的子包里使用 import 'virtual:uno.css';

### DISCUSSION THREAD

**Discussion by ggbond5277** [unocss/unocss#4330] [Q&A]
我现在有个`monorepo`项目，项目有两个包，分别是`package1`和`package2`。
`package1`是我的一个组件库，我在这个组件库里面使用了`import 'virtual:uno.css'`。
`pageage2`是我的普通项目，这个普通项目没有使用`unocss`，
我需要在这个`package2`普通项目里引用`package1`的组件，
现在我遇到了报错：
```bash
[vite] Internal server error: Failed to resolve import "virtual:uno.css" from "../package1/src/index.ts". Does the file exist?
```
这是我的一个[可以复现的项目](https://github.com/ggbond5277/monorepo-test)，
您可以在我的项目根目录执行`pnpm i` 再执行`pnpm run dev`
请问我该怎么解决这个问题

**Comment by zyyv** [comment]
你可以在 package2 使用 https://unocss.dev/integrations/cli 去扫描 package1 ，使用生成的后css.

**Reply by ggbond5277** [reply]
您好，这个是需要在`package2`项目下安装`@unocss/cli`才可以吧。
我不想对`package2`项目造成入侵。
因为我如果将`package1`项目构建成库模式
再在`package2`项目里引用由`package1`构建生成的`npm`包
此时`package2`是无需安装额外依赖的

**Reply by zyyv** [reply]
> 您好，这个是需要在package2项目下安装@unocss/cli才可以吧。

装在 root devDependencies 就行了吧

> 我不想对package2项目造成入侵。

不会的吧，它只是一行命令而已，看你如何使用了

**Reply by yaolunmao** [reply]
> 不会的吧，它只是一行命令而已，看你如何使用了

我研究研究 非常感谢
